coil/build.gradle

```diff
@@ -82,6 +82,7 @@ dependencies {
 
     implementation Libs.AndroidX.coreKtx
     implementation Libs.AndroidX.Compose.runtime
+    implementation Libs.AndroidX.Compose.material
```

Comment:
I'd rather not add this as a library dependency. Can we move this to be a test dependency instead?

Reply:
Done.